.github/workflows/continuous-integration-arm64.yml

```
@@ -1,18 +1,20 @@
-name: github-OSX
+name: github-arm64
 
 on:
   workflow_call:
 
 concurrency:
-  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-osx-x64
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-arm64
   cancel-in-progress: ${{github.event_name == 'pull_request'}}
 
 permissions: read-all
 
 jobs:
   osxci:
```

Do we know if this gets actually executed on a mac? Even if it ends up being the same arm architecture in linux, it still is a different processor and different os. I think we should keep an osx test as we advertise we can run on it.

```suggestion
  arm64ci:
```